content/docs/quickstart/go.md

```diff
@@ -3,98 +3,150 @@ title: Go Quick Start
 layout: quickstart
 short: Go
 description: This guide gets you started with gRPC in Go with a simple working example.
+protoc-version: 3.11.4
 ---
 
 ### Prerequisites
 
-- Go version 1.6 or higher.
-
-For installation instructions, see Go's [Getting Started](https://golang.org/doc/install) guide.
+- **Go** version 1.12+. For installation instructions, see Go's [Getting
+  Started](https://golang.org/doc/install) guide.
 
 #### gRPC
 
-Use the following command to install gRPC.
+Use the following command to install gRPC as a [module][]:
 
 ```sh
-$ go get -u google.golang.org/grpc
+$ export GO111MODULE=on # Enable module-aware mode
+$ go get -u google.golang.org/grpc@{{< param grpc_release_tag >}}
 ```
 
-#### Protocol Buffers v3
+#### Protocol Buffers
 
-Install the protoc compiler that is used to generate gRPC service code. The simplest way to do this is to download pre-compiled binaries for your platform(`protoc-<version>-<platform>.zip`) from here: [https://github.com/google/protobuf/releases](https://github.com/google/protobuf/releases)
+While not mandatory, gRPC applications usually leverage [Protocol Buffers][pb]
+for service definitions and data serialization, and the example code uses
+[proto3][].
 
-  * Unzip this file.
-  * Update the environment variable `PATH` to include the path to the protoc binary file.
+ 1. Install the `protoc` compiler:
 
-Next, install the protoc plugin for Go
+    - macOS:
 
-```sh
-$ go get -u github.com/golang/protobuf/protoc-gen-go
-```
+      ```sh
+      $ brew install protobuf
+      ```
 
-The compiler plugin, `protoc-gen-go`, will be installed in `$GOBIN`, defaulting
-to `$GOPATH/bin`. It must be in your `PATH` for the protocol compiler, protoc,
-to find it.
+    - Any OS:
 
-```sh
-$ export PATH=$PATH:$GOPATH/bin
-```
+       1. Download a zip file of the latest version of pre-compiled binaries for
+          your operating system from [github.com/google/protobuf/releases][]
+          (`protoc-<version>-<os><arch>.zip`). For example:
 
-### Download the example
+          ```sh
+          $ PB_REL="https://github.com/protocolbuffers/protobuf/releases"
+          $ curl -LO $PB_REL/download/v{{< param protoc-version >}}/protoc-{{< param protoc-version >}}-linux-x86_64.zip
+          ```
 
-The grpc code that was fetched with `go get google.golang.org/grpc` also contains the examples. They can be found under the examples dir: `$GOPATH/src/google.golang.org/grpc/examples`.
+       2. Unzip the file under `$HOME/local` or a directory of your choice. For
```

Comment:
@dfawley: I've changed `$HOME/local` to `$HOME/.local` (notice the leading dot). Any objections?

Reply:
Sounds fine - I don't have any strong feelings on this.